lib/src/network/backends.rs

```diff
@@ -158,7 +171,7 @@ impl BackendList {
 
   pub fn add_backend(&mut self, backend_id: &str, backend_address: &SocketAddr) {
     if self.backends.iter().find(|b| &(*b.borrow()).address == backend_address).is_none() {
-      let backend = Rc::new(RefCell::new(Backend::new(backend_id, *backend_address, self.next_id)));
+      let backend = Rc::new(RefCell::new(Backend::new(backend_id, *backend_address, self.next_id, 0)));
```

Comment:
why set the weight to 0 here? Can't we pass that parameter to `add_backend`?

Reply:
For the moment we don't use the `weight` so that was just a shortcut.